static/app/views/starfish/modules/APIModule/index.tsx

```diff
@@ -38,8 +39,10 @@ export default function APIModule(props: Props) {
         <Layout.Body>
           <Layout.Main fullWidth>
             <PageErrorAlert />
-            <APIModuleView {...props} onSelect={setSelectedRow} />
-            <EndpointDetail row={selectedRow} onClose={unsetSelectedSpanGroup} />
+            <PageFiltersContainer>
+              <APIModuleView {...props} onSelect={setSelectedRow} />
+              <EndpointDetail row={selectedRow} onClose={unsetSelectedSpanGroup} />
+            </PageFiltersContainer>
```

Comment:
Was wrapping this in the filtersContainer the fix for using the url stats period? I was going to look into fixing this for db too.

Reply:
Yup! Forgot how it works exactly, but `PageFiltersContainer` is the component responsible for parsing the url and then propagating the stats period to the page filter initialization